packages/hoppscotch-common/src/helpers/import-export/export/testResults.ts

```diff
@@ -0,0 +1,29 @@
+import { HoppTestResult } from "~/helpers/types/HoppTestResult"
+import { platform } from "~/platform"
+import * as E from "fp-ts/Either"
+
+export const ExportTestResults = async (testResults: HoppTestResult) => {
```

Comment:
we generally use `camel case` naming convention for function naming, you can change this to `exportTestResults`